My_project/Task_3.html

```diff
@@ -0,0 +1,203 @@
+            illo natus obcaecati odio perferendis qui, quisquam sapiente sint sit veritatis!</span><span>Ad animi aperiam
+            aspernatur atque debitis deserunt dolorem dolores dolorum ea eligendi et ipsum iusto magni molestias nam
+            nobis, non officiis perferendis quae sed sequi sit, tempore tenetur! Eaque, impedit.</span><span>Amet atque
+            aut dignissimos, eaque est ex expedita fugit illo in iure labore, maiores maxime officiis praesentium quisquam
+            quos repudiandae sit unde, vel voluptatibus! Cumque hic iusto laboriosam quasi similique?</span><span>A amet
+            architecto aut consectetur corporis cum dicta dignissimos dolorem est excepturi consectetur adipisicing elit.
+            exercitationem fuga illum ipsum iste libero mollitia natus nisi placeat praesentium quas quisquam quos, soluta
+            tempore vel voluptas.Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ab accusamus aut autem beatae
+            commodi consectetur dolores ducimus est, eveniet illo natus obcaecati odio perferendis qui, quisquam sapiente
+            sint sit veritatis!</span><span>Ad animi aperiam aspernatur atque debitis deserunt dolorem dolores dolorum ea
+            eligendi et ipsum iusto magni molestias nam nobis, non officiis perferendis quae sed sequi sit, tempore tenetur!
+            Eaque, impedit.</span><span>Amet atque aut dignissimos, eaque est ex expedita fugit illo in iure labore, maiores
+            maxime officiis praesentium quisquam quos repudiandae sit unde, vel voluptatibus! Cumque hic iusto laboriosam
+            quasi similique?</span><span>A amet architecto aut consectetur corporis cum dicta dignissimos dolorem est
+            excepturi exercitationem fuga illum ipsum iste libero mollitia natus nisi placeat praesentium quas quisquam quos,
+            soluta tempore vel voluptas.Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ab accusamus aut autem
+            beatae commodi consectetur dolores ducimus est, eveniet illo natus obcaecati odio perferendis qui, quisquam
+            sapiente sint sit veritatis!
+        </span>
+    </div>
+    <h2 style="text-align:center;text-transform: uppercase">upcoming courses</h2>
+    <div style="padding: 15px">
+        <hr>
+        <table style="background-color: #ffffff; width: 100%; border-collapse: collapse; font-weight: 800; table-layout: fixed">
+            <tr style="height: 90px">
+                <td style="border-bottom: 1px solid #808080">
+                    <span class="noLink"><strong>Программирование на GO</strong></span>
+                    <br>
+                    <span style="color:#181818; font-size: 12px">20.08.2020 - 20.12.2020</span>
+                    <br>
+                    <span style="color:#181818; font-size: 12px">Занятия вторник и четверг с 18:30 до 20:30</span>
+                </td>
+                <td style="max-width: calc(50% - 100px); border-bottom: 1px solid #808080">
```

Comment:
Why did you use `max-width` here? This column will stretch to needed width by default

Reply:
done